clients/fides-js/src/components/Overlay.tsx

```diff
@@ -21,22 +24,43 @@ export interface OverlayProps {
   experience: PrivacyExperience;
   cookie: FidesCookie;
   fidesRegionString: string;
+  modalLinkEl?: HTMLElement | null;
 }
 
 const Overlay: FunctionComponent<OverlayProps> = ({
   experience,
   options,
   fidesRegionString,
   cookie,
+  modalLinkEl,
 }) => {
   const [modalIsOpen, setModalIsOpen] = useState(false);
+  const [bannerIsOpen, setBannerIsOpen] = useState(false);
 
   if (!experience.experience_config) {
+    debugLog(options.debug, "No experience config found");
     return null;
   }
 
   const privacyNotices = experience.privacy_notices ?? [];
 
+  if (modalLinkEl) {
```

Comment:
Are you sure it's safe to call these kind of DOM methods here in a render function? This code will run many times (every render)

Reply:
^ good point, probably safest to put in a `useEffect`